pkg/providers/ovirt/validation/validators/storage-class-provider.go

```diff
@@ -0,0 +1,17 @@
+package validators
+
+import (
+	v1 "k8s.io/api/storage/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	storage "k8s.io/client-go/kubernetes/typed/storage/v1"
+)
+
+// StorageClasses is responsible for finding storage classes
+type StorageClasses struct {
```

Comment:
if that the only responsibility of it, perhaps `StorageClassFinder`
if this is expected to grow, no need to rename

Reply:
I don't think it will grow in the future - I will rename it.